src/lib/stitching/gravity/v2/stitching.ts

```diff
@@ -746,6 +758,18 @@ export const gravityStitchingEnvironment = (
             }
           },
         },
+        searchCriteriaConnection: {
```

Comment:
Rather than stitch this in, can be added to allow list [here](https://github.com/artsy/metaphysics/blob/10474f9f55bc52a940147aff7451d8ae6acdccad/src/lib/stitching/gravity/schema.ts#L11) and rely on schema merging.